crates/turborepo/tests/common/mod.rs

```diff
@@ -0,0 +1,56 @@
+use std::{path::Path, process::Command};
+
+use camino::Utf8Path;
+
+pub fn setup_fixture(
+    fixture: &str,
+    package_manager: Option<&str>,
+    test_dir: &Path,
+) -> Result<(), anyhow::Error> {
+    let script_path = Utf8Path::new(env!("CARGO_MANIFEST_DIR"))
+        .join("../../turborepo-tests/helpers/setup_integration_test.sh");
+
+    Command::new("bash")
+        .arg("-c")
+        .arg(format!(
+            "{} {} {}",
+            script_path,
+            fixture,
+            package_manager.unwrap_or("npm@10.5.0")
+        ))
+        .current_dir(test_dir)
+        .spawn()?
+        .wait()?;
```

Comment:
Can we capture `stdout`/`stderr` and make sure it gets into the error message? Would help a lot if this fails in CI and we want to know what went wrong.

Reply:
If the test fails, shouldn't cargo print stderr by default? And `spawn` forwards stderr, so it should show up in CI

Reply:
Ah you're right. My mistake.